package.json

```diff
@@ -13,5 +16,8 @@
       "type": "MIT",
       "url": "http://www.opensource.org/licenses/mit-license.php"
     }
-  ]
+  ],
+  "engines": {
+    "node": ">=6.4.0"
+  }
```

Comment:
I went with 6.4.0 here because I'm using `const` and and 6.4.0 is the first major version to support `const` fully. Without `const`, it needs to be 4.8.0 or higher because I'm using template strings.